collect_app/src/main/java/org/odk/collect/android/database/forms/FormDatabaseMigrator.java

```diff
@@ -70,13 +71,13 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion) throws SQLException {
                 upgradeToVersion11(db);
             case 11:
                 upgradeToVersion12(db);
-                break;
```

Comment:
> Is it possible these breaks could have been causing crashes/bugs?

Yes it was a bug so if someone was upgrading from lower versions the upgrade was not fully completed because of those breaks.